<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QMainWindow" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Инфо</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0,0,0,0">
                <property name="sizeConstraint">
                 <enum>QLayout::SetMaximumSize</enum>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Ваш никнейм:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="nickLine"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Адрес сервера:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="adressLine"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Порт</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="portLine"/>
                </item>
                <item>
                 <widget class="QPushButton" name="connectButton">
                  <property name="text">
                   <string>Подключиться</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Статус:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="statusLabel">
                    <property name="text">
                     <string>Отключен</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Статистика:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Сообщений отправлено:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="smsgLabel">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Файлов отправлено:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="sfileLabel">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>Сообщений получено:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="gmsgLabel">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QLabel" name="label_14">
                    <property name="text">
                     <string>Файлов получено:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="gfileLabel">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Чат</string>
           </attribute>
           <widget class="QWidget" name="verticalLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>731</width>
              <height>421</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string>Чат</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTextEdit" name="chatField"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_16">
                   <property name="text">
                    <string>Файлы</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="tableFiles">
                   <property name="editTriggers">
                    <set>QAbstractItemView::NoEditTriggers</set>
                   </property>
                   <property name="tabKeyNavigation">
                    <bool>false</bool>
                   </property>
                   <property name="showDropIndicator" stdset="0">
                    <bool>false</bool>
                   </property>
                   <property name="dragDropOverwriteMode">
                    <bool>false</bool>
                   </property>
                   <property name="selectionBehavior">
                    <enum>QAbstractItemView::SelectRows</enum>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>30</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="showGrid">
                    <bool>false</bool>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                   <property name="cornerButtonEnabled">
                    <bool>false</bool>
                   </property>
                   <attribute name="horizontalHeaderHighlightSections">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="horizontalHeaderMinimumSectionSize">
                    <number>100</number>
                   </attribute>
                   <attribute name="horizontalHeaderStretchLastSection">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="verticalHeaderVisible">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="verticalHeaderDefaultSectionSize">
                    <number>32</number>
                   </attribute>
                   <attribute name="verticalHeaderHighlightSections">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="verticalHeaderMinimumSectionSize">
                    <number>32</number>
                   </attribute>
                   <column>
                    <property name="text">
                     <string>Файл</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Имя файла</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Размер</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Отправитель</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="2,1,1">
               <item>
                <widget class="QLineEdit" name="messageLine"/>
               </item>
               <item>
                <widget class="QPushButton" name="messageButton">
                 <property name="text">
                  <string>Отправить сообщение</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="fileButton">
                 <property name="text">
                  <string>Отправить файл</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
